### PR TITLE
add horse class

### PR DESCRIPTION
**Pull request by ecchi1234**
check this code....

### CODE REVIEW

File: Horse.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+function Horse(name){
+    this.name = name;
+}
+module.exports = Horse;
```
Comment (ecchi123) [inline Horse.js:4]: change this line

Comment (ecchi1234) [reply Horse.js:4]: oke

File: index.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 console.log("hello word");
 console.log("ahii do ngoc!");
-var Cat = require('Cat');
+
+var Cat = require('./Cat');
 var tom = new Cat('tom', 'black');
```
Comment (ecchi123) [inline index.js:5]: ugly code

**Review by ecchi123 — COMMENTED** [review]
code is quite good

**Pull request merged**